subt/docker/sln_bash/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM osrf/subt-virtual-testbed:subt_solution_latest
```

Comment:
`zmq` is already installed in the parent image? I don't see it being installed anywhere.

Reply:
yes, it is part of the parent image - Ubuntu default?? (I am not sure) ... I reused what was there and only extended the linking by `-lzmq`